recipe/cache.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
I would rename the file to "opcache.php" as it is related to opcache, not cache in general.

Reply:
Also, shouldn't this file be required in sumo.php?

Reply:
Fixed